### ISSUE TITLE
Recent next: USB detect after FC reboot on OSX doesn't work

### ISSUE THREAD

**Issue by mlyle** [d-ronin/dRonin#1048]
Has to be a fairly recent regression.

**Comment by mlyle** [comment]
Or it's particular to some flight boards-- seen on Lux.

**Comment by mlyle** [comment]
Sparky2 does: 

```
"Telemetry connected"
"Telemetry disconnected"
Automatically opening device
ConnectionManager::devChanged autoconnected USB device
[RawHID] Error writing to device
```

**Comment by mlyle** [comment]
Looking at several targets-- the behavior is varied.
1. Sometimes usbmonitor sees the target go away and come back.  This is a race with our polling.  If usbmonitor sees this, all goes well.
2. Sometimes without that, ConnectionManager decides to try reconnect on its own.  Sometimes this works, sometimes RawHID gets an error writing to device which puts a stop to this.
3. Sometimes connectionmanager never tries.

**Comment by mlyle** [comment]
Fixed in #1081
